build/configs/rtl8721csm/loadable_apps/defconfig

```diff
@@ -550,6 +558,7 @@ CONFIG_UART2_2STOP=0
 # Wireless Device Options
 #
 CONFIG_DRIVERS_WIRELESS=y
+CONFIG_VIRTUAL_WLAN=y
```

Comment:
@pillip8282 , Could you check we can enable this by default?

Reply:
No it's not default. virtual_wlan is mocking driver to test wi-fi manager. So it have to be disabled in here.